src/main/java/frc4388/robot/subsystems/Location.java

```diff
@@ -0,0 +1,40 @@
+package frc4388.robot.subsystems;
+
+import frc4388.robot.subsystems.Apriltags;
+
+public class Location {
+    final Apriltags Apriltag = new Apriltags();
+
+    public boolean isLimelight = false;
+    public boolean isApriltag = false;
```

Comment:
These vars are internal (so far - high level design should get discussed later) so should not be exposed